modules/serverless-ossm-installing-and-configuring-openshift-serverless.adoc

```diff
@@ -57,31 +60,42 @@ kind: KnativeEventing
 metadata:
   name: knative-eventing
   namespace: knative-eventing
+  annotations:
+    serverless.openshift.io/disable-istio-net-policies-generation: "true"
 spec:
   config:
     features:
-      istio: enabled <1>
-  workloads: <2>
-  - name: pingsource-mt-adapter
-    labels:
-      "sidecar.istio.io/inject": "true"
-    annotations:
-      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
-  - name: imc-dispatcher
-    labels:
-      "sidecar.istio.io/inject": "true"
-    annotations:
-      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
-  - name: mt-broker-ingress
-    labels:
-      "sidecar.istio.io/inject": "true"
-    annotations:
-      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
-  - name: mt-broker-filter
-    labels:
-      "sidecar.istio.io/inject": "true"
-    annotations:
-      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+      istio: enabled  # <1>
+  workloads:  # <2>
+    - name: pingsource-mt-adapter
+      labels:
+        sidecar.istio.io/inject: "true"
+      annotations:
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+
+    - name: imc-dispatcher
+      labels:
+        sidecar.istio.io/inject: "true"
+      annotations:
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+
+    - name: mt-broker-ingress
+      labels:
+        sidecar.istio.io/inject: "true"
+      annotations:
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+
+    - name: mt-broker-filter
+      labels:
+        sidecar.istio.io/inject: "true"
+      annotations:
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+
+    - name: job-sink
+      labels:
+        sidecar.istio.io/inject: "true"
+      annotations:
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
 ----
 <1> Enable Eventing Istio controller to create a `DestinationRule` for each `InMemoryChannel` or `KafkaChannel` service.
 <2> Enable sidecar injection for Knative Eventing pods.
```

Comment:
```suggestion
* `spec.config.features.istio` enables Eventing Istio controller to create a `DestinationRule` for each `InMemoryChannel` or `KafkaChannel` service.
* `spec.workload` enables sidecar injection for Knative Eventing pods.
```
Refining this to follow the new style of [writing commands in code blocks](https://redhat-documentation.github.io/supplementary-style-guide/#explain-commands-variables-in-code-blocks).